scripts/verify-contracts.py

```diff
@@ -0,0 +1,68 @@
+import json
+from pathlib import Path
+import ape
+import click
+import structlog
+from eth_utils import to_checksum_address
+
+
+log = structlog.get_logger(__name__)
+
+
+def _get_contract_info(artifacts_dir: Path) -> dict[str, dict[str, dict[str, str]]]:
+    contract_info: dict[str, dict[str, dict[str, str]]] = {}
+    for file_path in artifacts_dir.glob("*.deployment.json"):
+        with open(file_path, "r") as fp:
+            data = json.load(fp)
+        chain_id = data["chain_id"]
+        if chain_id not in contract_info:
+            contract_info[chain_id] = {}
+        for contract_name, contract in data["contracts"].items():
+            if file_path.stem == "base.deployment":
+                ecosystem = "ethereum"
+            else:
+                _, ecosystem = file_path.stem.split(".")[0].split("-")
+            contract_info[chain_id][contract_name] = {
+                "address": contract["address"],
+                "ecosystem": ecosystem,
+                "network": file_path.parent.stem,
+            }
+
+    return contract_info
+
+
+def _get_rpc_info(rpc_file: Path) -> dict[str, str]:
+    with open(rpc_file, "r") as fp:
+        data = json.load(fp)
+    return data
```

Comment:
Now that the new deployment commands PR is merged, we should use new utilities to load RPC info and artifacts.

Reply:
done